VideoLocker/src/main/java/org/edx/mobile/module/prefs/UserPrefs.java

```diff
@@ -43,6 +46,14 @@ public File getDownloadFolder() {
         File packDir = new File(downloadsDir, context.getPackageName());
         File edxDir = new File(packDir, profile.username);
         edxDir.mkdirs();
+        try {
+            File noMediaFile = new File(edxDir, ".nomedia");
+            if (!noMediaFile.createNewFile()) {
+                logger.debug(".NoMedia file not created");
+            }
```

Comment:
Should we wrap this if-condition block inside
<code>
if ( ! noMediaFile.exists() ) 
</code>